devimint/src/federation.rs

```diff
@@ -139,15 +140,20 @@ impl Client {
             .unwrap())
     }
 
+    // TODO(support:v0.2): remove
```

Comment:
nit: I've seen it before and forgot to mention it, but "remove" here is not helpful at all. It would be more informative to say something 'in 0.3 we changed this and that' etc.

Reply:
Fixed, added more description to this comment